specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/vaults.json

```diff
@@ -928,6 +1007,172 @@
         },
         "upgradeDetails": {
           "$ref": "#/definitions/UpgradeDetails"
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateEndpointConnectionVaultProperties"
+          },
+          "description": "List of private endpoint connection.",
+          "readOnly": true
+        },
+        "privateEndpointStateForBackup": {
+          "description": "Private endpoint state for backup.",
+          "enum": [
+            "None",
+            "Enabled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "VaultPrivateEndpointState",
+            "modelAsString": true
+          }
+        },
+        "privateEndpointStateForSiteRecovery": {
+          "description": "Private endpoint state for site recovery.",
+          "enum": [
+            "None",
+            "Enabled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "VaultPrivateEndpointState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "IdentityData": {
+      "description": "Managed Identity Data, as received from ARM headers.",
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "description": "The tenant Id header.",
+          "type": "string"
+        },
+        "principalId": {
+          "description": "The principal Id header.",
+          "type": "string"
+        },
+        "type": {
+          "description": "The Identity type, can be SystemAssigned/UserAssigned/None.",
+          "type": "string"
+        }
+      }
+    },
+    "PrivateEndpointConnectionVaultProperties": {
```

Comment:
@deymadhumanti , could you please add description and readonly property for this as well?